content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -789,31 +789,40 @@ include metadata about the response.
 }
 ```
 
-The executed command is expected to print an `ExceCredential` to `stdout`. `k8s.io/client-go`
-will then use the returned bearer token in the `status` when authenticating against the
+The executed command is expected to print an `ExceCredential` to `stdout`.
+`k8s.io/client-go` will then use the returned bearer token and/or client TLS
+key and certificate from the `status` field when authenticating against the
 Kubernetes API.
 
+TLS credentials and bearer token are not mutually exclusive. Output with only
+`clientCertificateData` or only `clientKeyData` is an error.
+
 ```json
 {
   "apiVersion": "client.authentication.k8s.io/v1alpha1",
   "kind": "ExecCredential",
   "status": {
     "token": "my-bearer-token"
+    "clientCertificateData": "certificate PEM blob",
```

Comment:
make the examples look like
```json
"clientCertificateData":"-----BEGIN CERTIFICATE-----\n...\n-----END CERTIFICATE-----",
"clientKeyData":"-----BEGIN RSA PRIVATE KEY-----\n...\n-----END RSA PRIVATE KEY-----"
```
to make it clear how the content is passed/encoded

Reply:
Done